src/Utils/Apis/index.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
```

Comment:
Great job!

Reply:
You can optimise it even more


```suggestion
import axios from "axios";

const API_BASE_URL = 'https://burger-api-xcwp.onrender.com'

export const getPrices = async () =>
  await axios.get(API_BASE_URL + "/ingredients");

export const createOrder = async (orderData) =>
  await axios.post(API_BASE_URL + "/orders", orderData);

export const getContacts = async () =>
  await axios.get(API_BASE_URL + "/contact");

export const getOrders = async () =>
  await axios.get(API_BASE_URL + "/orders");
```